graphql/resolvers.go

```diff
@@ -2150,6 +2153,16 @@ func (r *taskQueueItemResolver) Requester(ctx context.Context, obj *restModel.AP
 	return TaskQueueItemTypeCommit, nil
 }
 
+func (r *mutationResolver) SaveProjectSettingsForSection(ctx context.Context, obj *restModel.APIProjectSettings, section string, isRepo bool) (*restModel.APIProjectSettings, error) {
```

Comment:
I'm afraid that the isRepo bool as opposed to a SaveProjectSettingsForSection and SaveRepotSettingsForSection (and same for the ProjectSettings resolver) will require a steeper learning curve for new engineers reading the code. It's not a huge difference but I feel like separating them makes the code a lot easier to follow.

Reply:
I'm a little anti duplicating code and they are pretty much identical. We can change it but then follow up work will need to follow the same convention. You'll be working with it the most though so whichever convention you prefer.

Reply:
Hahaha too much pressure! I'm ambivalent but it does feel little more straightforward to work two differently named functions like Chaya suggested. Maybe they can call the same util function with a boolean to avoid duplication? 😄